websocket/2.2/_index.md

```diff
@@ -9,35 +9,34 @@ This release removes references to the SecurityManager and provides some minor u
 
 
 ### New features, enhancements or additions
-* None
+* Clarify the responsibilities for sending ping and pong messages.
+
+* Add the `getSession()` method to `SendResult`.
+
+* Clarify the behaviour if `@OnMessage.maxMessageSize` it is set to a value larger than `Integer.MAX_VALUE`.
 
 ### Removals, deprecations or backwards incompatible changes
-* All references to the SecurityManager have been removed
+* None
```

Comment:
Should line 8 also be adjusted if the security manager hasn't been removed?

> This release removes references to the SecurityManager and provides some minor updates and clarifications.

Reply:
Thanks for spotting the inconsistency. This is an odd one. If you look at the code history, there were no references removed. However, the 2.1.1 release added a reference to the SecurityManager on the 2.1.x branch. From a user's perspective it does look like that reference has been removed so I'll restore the previous version of line 19 and keep line 8 as is.